integrated_channels/integrated_channel/transmitters/learner_data.py

```diff
@@ -309,7 +309,10 @@ def transmit(self, payload, **kwargs):  # pylint: disable=arguments-differ
                 enterprise_enrollment_id
             )
 
-            if not learner_data.course_completed:
+            if not learner_data.course_completed and (
+                not hasattr(self.enterprise_configuration, 'enable_incomplete_progress_transmission')
```

Comment:
why we are checking `enable_incomplete_progress_transmission` two times?

Reply:
will we send the data to integrated_channel when the learner has passed the course?

Reply:
> why we are checking `enable_incomplete_progress_transmission` two times?

@zamanafzal in test it is being tested twice so both the cases where if it is either set to true or false can be observed.

> will we send the data to integrated_channel when the learner has passed the course?
@zamanafzal well the approach was when this condition "not learner_data.course_completed" will be false the newly added checks won't affect current flow 
and so yes we should be able to send the data to integrated_channel when the learner has passed the course.

cc. @sameenfatima78

Reply:
I guess it doesn't make sense to add it two times.
we can use it like this
`if not learner_data.course_completed and not getattr(self.enterprise_configuration, 'enable_incomplete_progress_transmission', False):`

Reply:
@zamanafzal yes, it will be better to refactor it like you suggested.